src/modules/endpoint/reducers/initializeCityContent.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import type { CityContentStateType } from '../../app/StateType'
+import type { InitializeCityContentActionType } from '../../app/StoreActionType'
+
+const initializeCityContent = (
```

Comment:
The naming of this can stay initializeCityContent as it actually initialized it from an null/undefined state.